setupext.py

```diff
@@ -204,8 +203,16 @@ def check_CPP14_flags(possible_compile_flags):
 def check_for_pyembree(std_libs):
     embree_libs = []
     embree_aliases = {}
+
+    if sys.version_info >= (3, 9):
+        _path_finder = importlib.resources.files
+    else:
+        from pkg_resources import resource_filename
+        from functools import partial
+        _path_finder = partial(resource_filename, resource_name="rtcore.pxd")
+
     try:
-        _ = resource_filename("pyembree", "rtcore.pxd")
+        _path_finder("pyembree")
     except ImportError:
```

Comment:
I believe that `importlib.resources.files` raises `ModuleNotFoundError` instead of `ImportError` when the package in question is not installed. We need to expand that `except`.